frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -768,6 +769,7 @@ local VirtualKeyboard = FocusManager:new{
 }
 
 function VirtualKeyboard:init()
+    FocusManager.init(self)
```

Comment:
Re-initialize focus manager in case the keyboard was (dis)connected. That's necessary because VirtualKeyboard accesses fields `builtin_key_events` and `extra_key_events` on the DPad condition. If device has DPad, it assumes that those fields were initialized.